### ISSUE TITLE
[Business Approvals Kit - BUG] Unable to select the WorkflowProcess

### ISSUE THREAD

**Issue by ArunVRPowerDoc365** [microsoft/powercat-business-approvals-kit#163]
### Describe the issue

Create the Workflow in the App and try to select the workflow name in the Power Automate but i am unable to connect.
![image](https://github.com/microsoft/powercat-business-approvals-kit/assets/154434524/130edfbc-9005-41d0-8924-3b7691310ac4)

### What solution version are you using?

1.0.20231214.1

### What app or flow are you having the issue with?

Customer Connector

**Comment by Ramakrishnan24689** [comment]
@ArunVRPowerDoc365 - Can you please confirm if you have followed the suggested step provided [here](https://github.com/microsoft/powercat-business-approvals-kit/issues/155#issuecomment-1862099905)

**Comment by ArunVRPowerDoc365** [comment]
@Ramakrishnan24689 Yes, I followed the Step and the custom Connector show No error
![image](https://github.com/microsoft/powercat-business-approvals-kit/assets/154434524/788ca3a9-94a7-4049-bc80-2febf5ec63ee)
But as explained I created the flow in the New Solution and connect to Dataverse and add new step to custom connect, the Workflow Process is Showing NO Item

![image](https://github.com/microsoft/powercat-business-approvals-kit/assets/154434524/687ad4ce-b3a8-4c10-be80-418037b664da)

The Data is in the Dataverse Table 
![image](https://github.com/microsoft/powercat-business-approvals-kit/assets/154434524/d8313dcb-8cfa-4ef5-ab77-d623503fdee5)

**Comment by Ramakrishnan24689** [comment]
Thanks for confirmation @ArunVRPowerDoc365.
It appears as if the Process is not yet published.
Have you published the Approval process and do you see it Published Processes section of Approvals Designer site map as displayed in the below screen shot ?

![image](https://github.com/microsoft/powercat-business-approvals-kit/assets/11135056/bd12ed28-4246-4332-b421-3807c7e74813)

**Comment by ArunVRPowerDoc365** [comment]
@Ramakrishnan24689 
Yes the Status is Published 
![image](https://github.com/microsoft/powercat-business-approvals-kit/assets/154434524/857af762-6f8b-4578-8a67-669614357e11)
and Also the My submitted Workflow is always Empty

![image](https://github.com/microsoft/powercat-business-approvals-kit/assets/154434524/c878cd32-bfd2-44f3-be4e-eef46503f4d1)

**Comment by Ramakrishnan24689** [comment]
@ArunVRPowerDoc365 - It seems that the name is not saved in the process template you created.
<img width="1123" alt="image" src="https://github.com/microsoft/powercat-business-approvals-kit/assets/11135056/35f3f3a4-5996-415a-9b05-841f788ac1d3">

Can you edit the process by providing a name, and then proceed to save and publish ? 

<img width="938" alt="image" src="https://github.com/microsoft/powercat-business-approvals-kit/assets/11135056/20a26948-664b-4176-bf2a-5641ef205bf7">


Give it a try and let me know how that goes!

**Comment by ArunVRPowerDoc365** [comment]
@Ramakrishnan24689 

Yes, Saved the Process and then only I published. But Name is not saving
![image](https://github.com/microsoft/powercat-business-approvals-kit/assets/154434524/86d81996-093e-434e-b9f0-c716aaa02609)
![image](https://github.com/microsoft/powercat-business-approvals-kit/assets/154434524/ea3fe909-fbed-46d5-893f-2f5abe221b0c)

**Comment by ArunVRPowerDoc365** [comment]
@Ramakrishnan24689 I just tried the New Release, Still the name is not saving as i gave (Machine Request), but it saving as untitled. After 4 to 5 time Publish, it is saved in Given Name.

And unable to select the Select profile in Configure basic Setting. 

![image](https://github.com/microsoft/powercat-business-approvals-kit/assets/154434524/afed89d1-8ac4-4540-8489-f587cb82b3d4)

**Comment by craudales** [comment]
Hi. Currently I have the same original error. 
Is there a solution that you can share, please?
